Tests/SwiftPrettyPrintTests/Core/URLColoringWithDefaultColorThemeTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  URLColoringWithDefaultColorThemeTests.swift
+//  SwiftPrettyPrintTests
+//
+//  Created by ogasawara on 2021/02/08.
+//
+
+@testable import SwiftPrettyPrint
+import XCTest
+import SwiftParamTest
+import Curry
+
+/// Test the coloring of URL.
+/// - Note: The escape sequences used for the expected values of the test is as follows:
+///     - `\u{1B}`: escape characters starting all the escape sequences
+///     - `[0m`: Reset or normal. All attributes off
+///     - `[4m`: underline
+///     - `[33m`: make forground color yellow
+///     - `[34m`: make foreground color blue
+/// - SeeAlso:
+/// [ANSI escape code - Wikipedia](https://en.wikipedia.org/wiki/ANSI_escape_code)
+class URLColoringWithDefaultColorThemeTests: XCTestCase {
+
+    let describerWithDefaultColorTheme = PrettyDescriber(formatter: SinglelineFormatter(), theme: .default)
+    let exampleURL = "https://example.com"
+
+    override func setUpWithError() throws {}
+
+    override func tearDownWithError() throws {}
+
+    
+    /// URL coloring
+    /// - Throws: when the URL could not be generated from the string given in the `url` argument
+    func testURLColoring_non_debug() throws {
+        try _test(
+            describerWithDefaultColorTheme,
+            url: exampleURL,
+            expect: "\u{1B}[4m\u{1B}[34m\(exampleURL)\u{1B}[0m\u{1B}[4m\u{1B}[0m",
```

Comment:
The validity of ANSI-Escape string is guaranteed by [ColorlizeSwift](https://github.com/mtynior/ColorizeSwift).
Also this test-case is test `.default` theme only.

I suggest to create custom `ColorTheme` for tests.

For example:

```swift
let theme = {
    var t = ColorTheme.plain
    t.urlLiteral = { s -> "<URL>\(s)</URL>" }
    return t
}()
...
try _test(
    describerWithTestTheme,
    url: exampleURL,
    expect: "<URL>\(exampleURL)</URL>",
    debug: false
)
```

By doing this, test any `ColorTheme` works correctly (and test codes are clearly).

Reply:
resolve this in 5c229cd0ecc3c9a3887d971ce89e8b193fe2f8eb